news/dev.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed error in polyhedron volume caclulation for CN=5 and other polyhedrons with central atom on the same plane with equatorial atoms.
```

Comment:
Fixed -> Fix (Present tense for consistency)

<img width="614" alt="Screenshot 2025-06-18 at 10 02 46 PM" src="https://github.com/user-attachments/assets/33ea2e41-caf7-45a9-8dba-97a7ac5db75a" />

@dshirya please run pre-commit locally.. you can always set a shortcut like "pc" https://bobleesj.github.io/workflows.html#how-to-use-keyboard-shortcuts-in-your-cli